examples/rpc/starknet/starknet_l1NonceUnused.hurl

```diff
@@ -0,0 +1,13 @@
+POST {{RPC_URL}}
```

Comment:
What is this for? I'm not familiar with `.hurl` format.
`starknet_l1NonceUnused` neither with this rpc route. Where does it come from?

Reply:
Will remove the file as this is test for https://github.com/keep-starknet-strange/madara/pull/1205#discussion_r1403780855